lib/migrations/10-achievement-2.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
```

Comment:
Can you name this field with a word instead of `2`? Maybe `10-achievement-fields` would work

Reply:
Revised